ega-data-api-htsget/src/test/java/eu/elixir/ega/ebi/htsget/config/LocalTestData.java

```diff
@@ -0,0 +1,26 @@
+package eu.elixir.ega.ebi.htsget.config;
+
+public class LocalTestData {
+
+    public static final String BAM_FILE_ID = "EGAF00001770106";
+    public static final String BAM_INDEX_FILE_ID = "EGAF00001775034";
+
+    public static final String BAM_FILE_PATH = "/Users/afoix/mochi-git/ega-download-client/pyega3/EGAF00001770106/ENCFF000VWO.bam";
```

Comment:
I think we already discussed this, we probably need to sit together and concrete the details but this should be modified to a path that is accesible to our builder runner and any test that uses these files should be accessible through our gitlab builder.

Also the tests should be only run if a specific profile is present.

Reply:
@jorizci  this is for the next ticket that we are going to check deployment and testig.